src/main/java/org/openhab/binding/zwave/handler/ZWaveThingHandler.java

```diff
@@ -1331,6 +1376,9 @@ private void updateNodeProperties() {
             }
         }
 
+        properties.remove("zwave_lastheat");
```

Comment:
Type "zwave_lastheat". Should you use ZWaveBindingConstants.PROPERTY_LASTHEAL here?